src/expr.rs

```diff
@@ -428,7 +426,7 @@ fn rewrite_call(context: &RewriteContext,
         ends_with_newline: true,
     };
 
-    Some(format!("{}({})", callee_str, write_list(&items, &fmt)))
+    Some(format!("{}({})", callee_str, write_list(&items.collect::<Vec<_>>(), &fmt)))
```

Comment:
Usually, yes. I believe this is part of #133, too. Only when `ListTactic = HorizontalVertical` this will be problematic, as we'd need to walk the iterator an extra time to compute the width.